.gitignore

```diff
@@ -3,9 +3,9 @@
 .Rdata
 .httr-oauth
 .DS_Store
-
 #IDE related
 compile_commands.json
 .cache
 **/*.~undo-tree~
-*.~undo-tree~
\ No newline at end of file
+*.~undo-tree~
+.RData
```

Comment:
Missing *.Rproj. Added it in a commit, and deleted the file which is user-specific (make sure you back it up next time you pull, or get it using checkout from previous commit)